conda-store-server/conda_store_server/_internal/server/views/conda_store_ui.py

```diff
@@ -11,11 +11,16 @@
 
 
 @router_conda_store_ui.get("/")
+@router_conda_store_ui.get("/{full_path:path}")
 async def get_conda_store_ui(
     request: Request,
+    full_path: str,
     templates=Depends(dependencies.get_templates),
 ):
     context = {
         "request": request,
     }
-    return templates.TemplateResponse("conda-store-ui.html", context)
+    response = templates.TemplateResponse("conda-store-ui.html", context)
+    if full_path.endswith("not-found"):
+        response.status_code = 404
```

Comment:
Decided to sneak this one in to support https://github.com/conda-incubator/conda-store-ui/pull/440

Reply:
Perfect time for it :rocket: